CHANGES.rst

```diff
@@ -367,6 +369,8 @@ New Features
 
 - ``astropy.vo``
 
+  - KeyError when converting Table v1.2 numeric arrays fixed. [#4781]
```

Comment:
Since both releases are marked as unreleased how is one to know which is
the duplicate? Obviously I can tell by the line numbers, but I am
suggesting that having 1.2 and 1.1.3 both as unreleased makes it very
unclear as to which release is going to include the pull release. Hence
I put it in both.

On Thu, 2016-04-14 at 13:30 -0700, P. L. Lim wrote:
> In CHANGES.rst:
> 
> > @@ -367,6 +369,8 @@ New Features
> >  
> >  - ``astropy.vo``
> >  
> > +  - KeyError when converting Table v1.2 numeric arrays fixed. [#4781]
> 
> Remove duplicate change log entry.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> 
> 

-- 
Al Niessner

I have never found the companion that was so companionable as solitude.
- From Walden by Henry David Thoreau

The universe is indifferent, and life is brutal; however, it is man's
choice of behavior that makes them malevolent rather than benevolent.

Some will fall in love with life and drink it from a fountain
That is pouring like an avalanche coming down the mountain.
- From the song Pepper by the Butthole Surfers